homeassistant/components/airly/sensor.py

```diff
@@ -80,15 +80,15 @@ class AirlySensorEntityDescription(SensorEntityDescription):
     AirlySensorEntityDescription(
         key=ATTR_API_PM1,
         device_class=SensorDeviceClass.PM1,
-        name=ATTR_API_PM1,
+        name="Particulate matter 1.0",
```

Comment:
These names are pretty long, wouldn't PM 2.0 be easier to deal with in the UI?

Reply:
Looking at several other air quality integrations, they generally seem to name the sensor PM1.0, PM2.5, etc. (no space) so seems like that would be the most consistent way.

Reply:
As suggested, I changed these names. I also reverted `CAQI` (instead of `Common Air Quality Index`) because that name is also very long.